lib/matplotlib/axes/_axes.py

```diff
@@ -2809,11 +2809,19 @@ def pie(self, x, explode=None, labels=None, colors=None,
         The axes aspect ratio can be controlled with `Axes.set_aspect`.
         """
         self.set_aspect('equal')
-        x = np.array(x, np.float32)
+        # The use of float32 is "historical", but can't be changed without
+        # regenerating the test baselines.
+        x = np.asarray(x, np.float32)
```

Comment:
Could be changed to float64, and only patched back in some way to float32 for the tests. But that's beyond this PR.

Reply:
seems not worth the effort, frankly